gravitee-am-identityprovider/gravitee-am-identityprovider-github/src/main/java/io/gravitee/am/identityprovider/github/GithubIdentityProviderConfiguration.java

```diff
@@ -100,6 +101,14 @@ public void setConnectTimeout(Integer connectTimeout) {
         this.connectTimeout = connectTimeout;
     }
 
+    public Integer getIdleTimeout() {
```

Comment:
The idleTimeout is never set to the WebClientOptions

https://github.com/gravitee-io/gravitee-access-management/blob/issues%2F%233505-http-request-timeout-social-idp/gravitee-am-identityprovider/gravitee-am-identityprovider-github/src/main/java/io/gravitee/am/identityprovider/github/authentication/spring/GithubAuthenticationProviderConfiguration.java#L55

Reply:
Changes applied, thanks.

Reply:
Done